src/blas/backends/cublas/cublas_level3.cpp

```diff
@@ -95,12 +95,12 @@ inline void gemm_ex(DATATYPE_A DT_A, DATATYPE_B DT_B, DATATYPE_C DT_C, sycl::que
             auto c_ = sc.get_mem<cuDataType_C*>(c_acc);
             cublasStatus_t err;
 #ifdef SYCL_EXT_ONEAPI_ENQUEUE_NATIVE_COMMAND
-            CUBLAS_ERROR_FUNC_SYNC(cublasGemmEx, err, handle, get_cublas_operation(transa),
-                                   get_cublas_operation(transb), m, n, k, (cuDataType_C*)&alpha, a_,
-                                   DT_A, lda, b_, DT_B, ldb, (cuDataType_C*)&beta, c_, DT_C, ldc,
-                                   DT_C, CUBLAS_GEMM_DEFAULT);
-#else
             CUBLAS_ERROR_FUNC(cublasGemmEx, err, handle, get_cublas_operation(transa),
+                              get_cublas_operation(transb), m, n, k, (cuDataType_C*)&alpha, a_,
+                              DT_A, lda, b_, DT_B, ldb, (cuDataType_C*)&beta, c_, DT_C, ldc, DT_C,
+                              CUBLAS_GEMM_DEFAULT);
+#else
+            CUBLAS_ERROR_FUNC_SYCN(cublasGemmEx, err, handle, get_cublas_operation(transa),
```

Comment:
```suggestion
            CUBLAS_ERROR_FUNC_SYNC(cublasGemmEx, err, handle, get_cublas_operation(transa),
```

Reply:
woops, thanks for spotting that. I did an AdaptiveCpp build locally and it caught that this didn't compile, so I'll try to build with that to double check PRs going forward.

Reply:
Thanks for reviewing this patch @andrewtbarker , do you have any more feedback or can you approve the PR?